templates/nvidia-guardrails-with-RAG/README.md

```diff
@@ -0,0 +1,130 @@
+# Guardrails Integration with LangChain Expression Language Template
+
+This example demonstrates integrating NeMo Guardrails with LangChain Expression Language Templates. Here we start of with a `nvidia-rag-canonical` from the LangChain [templates](https://github.com/langchain-ai/langchain/tree/master/templates/nvidia-rag-canonical) which performs RAG using Milvus Vector Store and NVIDIA Models, both embedding and Chat.
+
+## Environmental Setup
+
+### Exporting API keys as Environment Variables
+
+To start working with this example, you should start with exporting NVIDIA API Key and NGC API Key as environment variables. To create the NVIDIA API Key, you can create one by following these steps:
+
+1. Create a free account with the NVIDIA GPU Cloud service, which hosts AI solution catalogs, containers, models, etc.
+
+2. Navigate to `Catalog > AI Foundation Models >  (Model with API endpoint).`
+
+3. Select the `API` option and click `Generate Key`.
+
+4. Save the generated key as `NVIDIA_API_KEY`. From there, you should have access to the endpoints.
+
+To create the NGC API Key, you can create one by following these steps:
+
+1. create a free account with the [NVIDIA NGC Catalog](https://ngc.nvidia.com/signin). 
+
+2. Navigate to `Setup > Generate API Key` and follow the steps.
+
+3. Save the generated key as `NGC_API_KEY`. From there, you should have access to NGC models
+
+```
+export NVIDIA_API_KEY=<your nvidia_api_key>
+export NGC_API_KEY=<your ngc_api_key>
+```
+### NeMo Guardrails
+
+To add guardrails, first you need to clone the nemoguardrails package with langchain runnable integrations as follows:
+
+Create a directory named `nemoguardrails`, clone and install the package with all necessary dependencies
```

Comment:
This is a bit heavyweight for a template

Reply:
@efriis a simple 'pip install nemoguardrails' should work. All the latest modules needed here have been updated on the guardrails too. Please let me know how I can make this change in the README.md file